Use pytest option "-s"  shortcut for "--capture=no" in foreman-maintain-automation.

We are going to use [fabric](http://docs.fabfile.org/en/2.5/) in Testfm and were facing issue[1] during test execution. After few trial and error, searching over internet for solution we have found that the '-s' option needs to be used to solve the problem.

[1] `OSError: reading from stdin while output is captured`
```
        if thread_exceptions:
>           raise ThreadException(thread_exceptions)
E           invoke.exceptions.ThreadException: 
E           Saw 1 exceptions within threads (OSError):
E           
E           
E           Thread args: {'kwargs': {'echo': None,
E                       'input_': <_pytest.capture.DontReadFromInput object at 0x7f74032e6f60>,
E                       'output': <_pytest.capture.EncodedFile object at 0x7f74032e6fd0>},
E            'target': <bound method Runner.handle_stdin of <fabric.runners.Remote object at 0x7f73feb5ada0>>}
E           
E           Traceback (most recent call last):
E           
E             File "/home/jenkins/shiningpanda/jobs/7271201b/virtualenvs/d41d8cd9/lib/python3.6/site-packages/invoke/util.py", line 233, in run
E               super(ExceptionHandlingThread, self).run()
E           
E             File "/usr/lib64/python3.6/threading.py", line 864, in run
E               self._target(*self._args, **self._kwargs)
E           
E             File "/home/jenkins/shiningpanda/jobs/7271201b/virtualenvs/d41d8cd9/lib/python3.6/site-packages/invoke/runners.py", line 687, in handle_stdin
E               data = self.read_our_stdin(input_)
E           
E             File "/home/jenkins/shiningpanda/jobs/7271201b/virtualenvs/d41d8cd9/lib/python3.6/site-packages/invoke/runners.py", line 646, in read_our_stdin
E               bytes_ = input_.read(bytes_to_read(input_))
E           
E             File "/home/jenkins/shiningpanda/jobs/7271201b/virtualenvs/d41d8cd9/lib/python3.6/site-packages/_pytest/capture.py", line 591, in read
E               raise IOError("reading from stdin while output is captured")
E           
E           OSError: reading from stdin while output is captured

../../shiningpanda/jobs/7271201b/virtualenvs/d41d8cd9/lib/python3.6/site-packages/invoke/runners.py:399: ThreadException
----------------------------- Captured stdout call -----------------------------
```

--- a/scripts/foreman-maintain-automation.sh
+++ b/scripts/foreman-maintain-automation.sh
@@ -31,7 +31,7 @@ if [[ "$TEST_UPSTREAM" = "true" ]]; then
         ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@${SERVER_HOSTNAME} 'cd foreman_maintain; git fetch origin pull/'${PR_NUMBER}'/head:'${BRANCH_NAME}'; git checkout '${BRANCH_NAME}
     fi
 fi
-if [[ "$SATELLITE_VERSION" != "6.3" ]] || [[ "$SATELLITE_VERSION" != "6.4" ]]; then
+if [[ "$SATELLITE_VERSION" != "6.3" ]] || [[ "$SATELLITE_VERSION" != "6.4" ]] || [[ "$TEST_UPSTREAM" = "false" ]]; then
     sed -i "s/foreman-maintain {0} {1} {2}/satellite-maintain {0} {1} {2}/g" testfm/base.py
 fi
 export ANSIBLE_HOST_KEY_CHECKING=False
@@ -40,11 +40,11 @@ if [ "${COMPONENT}" == "capsule" ]; then
 fi
 set +e
 if [ -n "${PYTEST_OPTIONS}" ]; then
-    pytest -v --junit-xml=foreman-results.xml --ansible-host-pattern "${COMPONENT}" --ansible-user root --ansible-inventory testfm/inventory ${PYTEST_OPTIONS}
+    pytest -sv --junit-xml=foreman-results.xml --ansible-host-pattern "${COMPONENT}" --ansible-user root --ansible-inventory testfm/inventory ${PYTEST_OPTIONS}
 elif [ -n "${PYTEST_MARKS}" ]; then
-    pytest -v --junit-xml=foreman-results.xml --ansible-host-pattern "${COMPONENT}" --ansible-user root --ansible-inventory testfm/inventory tests/ -m "${PYTEST_MARKS}"
+    pytest -sv --junit-xml=foreman-results.xml --ansible-host-pattern "${COMPONENT}" --ansible-user root --ansible-inventory testfm/inventory tests/ -m "${PYTEST_MARKS}"
 else
-    pytest -v --junit-xml=foreman-results.xml --ansible-host-pattern "${COMPONENT}" --ansible-user root --ansible-inventory testfm/inventory tests/
+    pytest -sv --junit-xml=foreman-results.xml --ansible-host-pattern "${COMPONENT}" --ansible-user root --ansible-inventory testfm/inventory tests/
 fi
 set -e
 if [[ "$TEST_UPSTREAM" != "true" ]]; then

